resources/cluster-essentials/charts/delete-helm-release/templates/xip-patch.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "delete-helm-release.fullname" . }}-xip-patch-job
+  namespace: kyma-installer
```

Comment:
```suggestion
  namespace: {{ .Release.Namespace }}
```

Reply:
This won't work. "cluster-essentials" is installed in kyma-system, but the job should work in "kyma-installer" namespace, because it uses the same service-account as the Installer. This is the "allmighty" SA, because uninstalling xip-patch requires removing objects in several namespaces.

Reply:
I understand, in this case, IMHO, it would be best to create a new SA in this chart, which will use the reader ClusterRole, to avoid multiple namespaces